source/appendices/e-audit-requirements-and-tests/02-audit-tests.rst

```diff
@@ -5,1045 +5,391 @@
+The auditor MUST request
 
-* Each book must have 5 investigations of unique items. This will report 5 Total_Item_Investigations, 5 Unique_Item_Investigations, and 1 Unique_Title_Investigations.
+* 50 journal articles with Access_Type Open.
+* 50 journal articles with Access_Type Free_To_Read.
 
-This must result in:
+This MUST result in 50 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests with Access_Type Open, and the same again for Access_Type Free_To_Read.
 
-* 25 OA_Gold Total_Item_Investigations and 25 Controlled Total_Item_Investigations being reported in the TR_B3 Standard View.
-* 25 OA_Gold Unique_Item_Investigations and 25 Controlled Unique_Item_Investigations being reported in the TR_B3 Standard View.
-* 5 OA_Gold Unique_Title_Investigations and 5 Controlled Unique_Title_Investigations being reported in the TR_B3 Standard View.
 
-**Option 2**: Content provider does not offer OA_Gold items.
+E.2.5.2 Journal Year of Publication
+'''''''''''''''''''''''''''''''''''
 
-The auditor must make a total of 50 Investigations within a subset of books.
+For journal content, year of publication (YOP) is useful in evaluating usage of archive content.
 
-* Each book must have 5 investigations of unique items. This will report 5 Total_Item_Investigations, 5 Unique_Item_Investigations, and 1 Unique_Title_Investigations.
+The auditor MUST confirm the Year of Publication (YOP) of articles covered in other Title Report audit tests with appropriate and proportionate spot checks covering a minimum of 10% of all Title Report Audit Tests: Journals.
 
-This must result in:
+If the YOP appearing in the reports is different from that of the journal article for more than 10% of the checked items, the auditor must expand their spot checks to cover at least 25% of tested journal articles. If 10% or more of the journal articles have a different YOP from that in the reports, the report provider has failed the Journal YOP audit test.
 
-* 50 Controlled Total_Item_Investigations being reported in the TR_B3 Standard View.
-* 50 Controlled Unique_Item_Investigations being reported in the TR_B3 Standard View.
-* 10 Controlled Unique_Title_Investigations being reported in the TR_B3 Standard View.
 
+E.2.6 Item Report Audit Tests
+"""""""""""""""""""""""""""""
 
-Audit Test B3-4: Total_Item_Investigations, Unique_Item_Investigations, and Unique_Title_Investigations - 30-second filters
-...........................................................................................................................
+The Item Report will be COUNTER-compliant if the following audit tests are passed. The Standard Views of the Item Report (IR_A1 and IR_M1) will be COUNTER-compliant if the metrics match those in the Item Report.
```

Comment:
If Components are supported, there would be an aggregation required for comparing the Total_Item counts in IR_A1 and IR_M1 with IR, or the Components would have to be omitted from the test?